.ruby-version

```diff
@@ -1 +1 @@
-2.5.1
+2.6.6
```

Comment:
prod container uses `ruby-2.6.3-106.module+el8.1.0+3653+beb38eb0.x86_64`, would this change affect it?

Reply:
Not really, just local deployment. The container is always influenced by what is in the dockerfile.

Reply:
We should probably add this to .gitignore. I'm not sure anyone on our team even uses rbenv.